frontend/apps/crates/entry/jig/play/src/player/sidebar/dom/like.rs

```diff
@@ -0,0 +1,40 @@
+use std::rc::Rc;
+
+use dominator::{clone, html, Dom};
+
+use shared::{
+    api::{endpoints::jig, ApiEndpoint},
+    error::EmptyError, domain::jig::JigResponse,
+};
+
+use utils::{
+    prelude::api_with_auth_empty,
+    events
+};
+
+use super::super::state::State;
+
+pub fn render(state: Rc<State>, jig: &JigResponse) -> Dom {
+    html!("jig-play-sidebar-action", {
+        .property("slot", "actions")
+        .property("kind", "like")
+        // TODO Render active or not active
+        .event(clone!(state, jig => move |_: events::Click| {
+            state.loader.load(async move {
```

Comment:
Something to consider but might not be a problem at all:
loader can only load one future at a time, so if you call `loader.load` when something is already loading the first one will be dropped, so make sure that it's not used anywhere else that could potentially overlap here.

Now that I think about it, this problem could cause some very real issues, we should try coming up with something better (later).

Reply:
@MendyBerger I saw on the profile page when editing details that running requests are cancelled when a new one is made. I had (incorrectly) assumed that it would only cancel the same type of request.

I agree. Perhaps as an interim solution we can remove the [abort call](https://github.com/dakom/dominator-helpers/blob/master/src/futures.rs#L42) from AsyncLoaders replace method?